### DISCUSSION TITLE
Scrolling

### DISCUSSION THREAD

**Discussion by ElTagoury** [moudey/Shell#345] [Ideas]
I hoped for Microsoft to do it in Windows 11's new Context menu but they didn't, i wish if there is any tool that gives the ability to scroll long context menu's but unfortunately there isn't, I'm very happy of knowing and using Nilesoft it's a great tool & i hope if they can add this feature and be able to scroll through Windows Explorer's Context Menu.
Good Effort & Much Appreciated.

**Comment by moudey** [comment]
Thank you for your interest. 
The scrolling issue is being developed. You can use the up and down keys to avoid this issue.
